### PR TITLE
Use hash sums for WIX file ids

### PR DESCRIPTION
**Pull request by pozdnyakov**
This prevents from generating too long file ids that are not supported by WIX.

BUG=XWALK-5679

### CODE REVIEW

**Comment by pozdnyakov** [comment]
@r0b5t4 PTAL

File: windows/lib/WixSDK.js [hunk 1]
```diff
@@ -148,9 +149,18 @@ function(app_path, xwalk_path, meta_data, callback) {
     var file_ids = [];
 
     function MakeIdFromPath(path) {
-        // Only ASCII, digits, '.', '_' are allowed.
-        return path.replace(new RegExp('[^A-Za-z0-9_.]', 'g'), '_');
+        shasum = crypto.createHash('sha1');
```
Comment (r0b5t4) [inline windows/lib/WixSDK.js]: var shasum?

Comment (pozdnyakov) [inline windows/lib/WixSDK.js]: https://nodejs.org/api/crypto.html#crypto_hash_update_data_input_encoding
naming is taken from ^

Comment (pozdnyakov) [inline windows/lib/WixSDK.js]: ah, you mean 'var'.. but I wanted to keep it global

Comment (r0b5t4) [inline windows/lib/WixSDK.js]: But does it make any difference? crypto.createHash() is still run on every invocation of MakeIdFromPath(), right? In that case I'd prefer not to have it global.

**Pull request merged**